cb-talent-development/employer-style-base
# EM-1492: Small Biz Wrapup

**Purpose:**
> As a developer, I would like to pull all page elements together, so the page functions as intended.

**Adds**
 - `toggle` mixins
 - `.screen-reader-only` class
 - refactors section header mixins and adds `--subhead` modifier
 - removes `table--pricing-plan`, we only need `table--product`

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/EM-1492

**Changes:**
* Changes to setup, Architectural changes, Migrations, Side effects, Library changes
  * n/a

**QA Links:**
Before: http://web.employer-3.development.c66.me/recruiting-solutions/small-business-subscription-plans-and-pricing
After: http://web.employer-7.development.c66.me/recruiting-solutions/small-business-subscription-plans-and-pricing

**How to Verify These Changes**
* Specific pages to visit
  * http://web.employer-7.development.c66.me/recruiting-solutions/small-business-subscription-plans-and-pricing

* Steps to take
  * see https://github.com/cbdr/employer/pull/879

**Relevant PRs/Dependencies:**
https://github.com/cbdr/employer/pull/879

**Additional Information**
